### ISSUE TITLE
Unsupported device: dmaker.fan.p45

### ISSUE THREAD

**Issue by aaastafyev** [syssi/xiaomi_airpurifier#343]
Please add device support.
This is a new model and it is becoming very popular.

**Comment by biscuitee** [comment]
Yes, i am also using dmaker.fan.p45 tower fan.  Appreciate if it can be supported soon.  Currently i added it as dmaker.fan.p9 and it can only control the on/off toggle, while the speed control are not working.   Appreciate if dmaker.fan.p45 can be officially supported.
